holocron_api/api/models/standard.py

```diff
@@ -4,12 +4,26 @@
 from Ad_Network import Ad_Network
 
 
+class Program(models.Model):
+    program_key = models.CharField(max_length=100, primary_key=True)
```

Comment:
I think Program should have an "id" not a key. There's probably little value in making someone come up with a "key"... an auto-number ID is probably better.